utils/index.js

```diff
@@ -251,13 +262,22 @@ const hasData = (obj, key, returnVal = false) => {
   return bool
 }
 
-const currencyFilter = (number, fractionDigits = 2) => {
+const currencyFilter = (number, locale = 'en', fractionDigits = 2) => {
+
   const amount = Number(number)
 
-  return amount.toLocaleString('en-US', {
+  const localeSetting = (locale === 'en') ? 'en-US' : 'fr-CA'
+
+  const filteredAmount = amount.toLocaleString(localeSetting, {
     minimumFractionDigits: fractionDigits,
     maximumFractionDigits: fractionDigits,
   })
+
+  if(locale === 'fr') {
+    return `${filteredAmount}$`
+  }
+
+  return `$${filteredAmount}`
```

Comment:
This is the same function as in the cypress tests, right? Can we just use this one or is there a reason to have 2?

(also, let's remove `fractionDigits`)

Reply:
For whatever reason I could access this from the Cypress tests. Maybe I was just doing something wrong, but it seemed like you can't access outside of the cypress folder :/ 
I'll take another look though, because I also don't like duplicating it.